.github/workflows/main.yaml

```diff
@@ -26,6 +26,11 @@ jobs:
     steps:
     - name: Checkout code
       uses: actions/checkout@v2
+    - uses: actions/setup-go@v2
+      with:
+        go-version: '1.17'
+   - name: tidy
+     run: go mod tidy
```

Comment:
Not sure if this is an issue but these 2 lines do not line up.

Reply:
Fixed